app/assets/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,26 @@
+.card {
+  width: 12rem;
+  margin: 0 1rem;
+  text-align: center;
+  border-radius: 0.5rem;
+
+  .card-body {
```

Comment:
we usually use a double underscore to separate the element class name (card) & block name (body) and you don't need to repeat the class name when using nesting. So the above code could be written as
```scss
.card
  ...

  &__body {
  ...
  }
```
more examples could be found on [Compass](https://compass.nimblehq.co/development/code-conventions/css/#class-names)

Reply:
Fixed in 631491778f1ba7a60beaebc3728fa9d8ed0dd09f

Reply:
I fixed on the wrong branch, Sorry for this sloppy.
Will make the change

Reply:
Fixed in da667a03545256a083fd71aa9492f621f14186e6